tests/neg/17284.scala

```diff
@@ -0,0 +1 @@
+def test = 451.synchronized {}
```

Comment:
This test should be testes with fatal warnings

Reply:
Yes, I committed an old version of the test file by mistake. Fixed.